test/test_build.py

```diff
@@ -93,6 +94,13 @@ def test_build_parallel(self):
         self.assertEqual(output, 'done')
 
     def test_read_config(self):
+        class _DummyTask(gokart.TaskOnKart):
```

Comment:
fixes a bug unrelated to the main changes:

The function `gokart.utils.add_config(config_file_path)` was altering the default values of Task, which affected the outcomes of other tests.
This problem has been resolved by implementing a dedicated task for it.